packages/text-editor/src/AudioRecorder.ts

```diff
@@ -0,0 +1,57 @@
+export default class AudioRecorder {
```

Comment:
@nsklyarov 
Could you please add Header with License. Thanks.

Reply:
Sure! Added!